src/components/Navbar/Navbar.js

```diff
@@ -0,0 +1,82 @@
+import { StaticImage } from 'gatsby-plugin-image';
```

Comment:
przeformatuj dla pewności przez ctrl+shift+p format document (forced) zobacz czy masz domyślny formater na prettiera

Reply:
powinienem miec bo zmienialismy to z kamilem
I czy nadal mam te zmiany wprowadzić jak juz z mergowaliscie?